README.md

```diff
@@ -448,6 +448,12 @@ GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE=1 # specific to Netlify, for large sites
 GATSBY_CPU_COUNT=2 # limits the number of Gatsby threads, helping with deployment stability
 NODE_VERSION=18 # this is required by Gatsby v5
 NODE_OPTIONS=--max-old-space-size=4096 # increase default heap size to prevent crashes during build
+# The following "CLOUDFLARE_R2" variables are required to create the /research/snapshots/ page
+CLOUDFLARE_R2_ACCOUNT_ID=[The Cloudflare R2 account ID (ie: 8f4144a9d995a9921d0200db59f6a00e)]
```

Comment:
nitpick: i.e. -> e.g., here and below :-)

Reply:
Done!